proposals/lip-0045.md

```diff
@@ -655,110 +688,59 @@ An entry for the mainchain is present by default, where:
 
 The `appendToInboxTree` function appends a new element to the underlying Merkle tree of the inbox of a chain account.
 
-##### Parameters
-
-This function has the following input parameters in the order given below:
-
-* `chainID`: The partner chain ID.
-* `appendData`: A bytes value.
-
-##### Returns
-
-A boolean indicating the success of the function execution.
-
-##### Execution
-
 ```python
-appendToInboxTree(chainID, appendData):
+def appendToInboxTree(chainID: ChainID, appendData: bytes) -> None:
     # Update root, appendPath, and size
-    let inboxTree be the underlying Merkle tree of channel(chainID).inbox
+    inboxTree = Merkle tree of channel(chainID).inbox
     inboxTree.append(SHA-256(appendData))
     channel(chainID).inbox.root = inboxTree.root
     channel(chainID).inbox.appendPath = inboxTree.appendPath
     channel(chainID).inbox.size = inboxTree.size
-
-    return True
 ```
 
 #### appendToOutboxTree
 
 The `appendToOutboxTree` function appends a new element to the underlying Merkle tree of the outbox of a chain account.
 
-##### Parameters
-
-This function has the following input parameters in the order given below:
-
-* `chainID`: The partner chain ID.
-* `appendData`: A bytes value.
-
-##### Returns
-
-A boolean indicating the success of the function execution.
-
-##### Execution
-
 ```python
-appendToOutboxTree(chainID, appendData):
+def appendToOutboxTree(chainID: ChainID, appendData: bytes) -> None:
     # Update root, appendPath, and size
-    let outboxTree be the underlying Merkle tree of channel(chainID).outbox
+    outboxTree = Merkle tree of channel(chainID).outbox
```

Comment:
Here I assume the whole Merkle tree is not necessary, rather the append path, root and size is sufficient. Maybe this could be made more clear.

Reply:
Similar suggestion as above.

Reply:
fixed